src/bridge/BridgeBase/index.ts

```diff
@@ -24,7 +24,8 @@ export class BridgeBase extends LitElement {
   @property({ attribute: false, type: String }) validateMessage;
   @property({ attribute: false, type: Object }) theComponentStyleTemplate: any = nothing; // Any styling lit-html template that should be added to renderTemplates
 
-  @property({ attribute: false, type: Object }) thePConn; // Normalize incoming pConn to a PConnect object
+  // @ts-ignore
+  @property({ attribute: false }) thePConn: typeof PConnect; // Normalize incoming pConn to a PConnect object
```

Comment:
Manasa, The type of the "pConn" below needs to be defined too since we're using that also in the components and without it being defined, typings won't work in those cases.

Reply:
pConn below is not of type PConnect and we are not using much in the components.

Reply:
We should only be referring to thePConn as it goes through normalization (for some reason I still don't quite understand).  Any references to this.pConn are likely incorrect outside of BridgeBase.

Reply:
Yeah, it's quite confusing also. We need to remove these occurrences probably in the future release.